pilot/pkg/proxy/agent_test.go

```diff
@@ -143,6 +143,7 @@ func TestApplyThrice(t *testing.T) {
 		if epoch == 1 {
 			go func() {
 				a.ConfigCh() <- good
+				time.Sleep(time.Millisecond * 10)
```

Comment:
Is this enough of a delay? I know prow has been problematic with timing issues. Maybe just make it a second?

Reply:
Yeah, making it 1s would be much safer.